sdk/translation/Azure.AI.Translation.Document/samples/CodeSamples/SampleAuthentication.cs

```diff
@@ -0,0 +1,38 @@
+﻿// <copyright file="SampleAuthentication.cs" company="Microsoft Corporation">
+// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+using Azure;
+using Azure.AI.Translation.Document;
+using System;
+using System.Threading.Tasks;
+
+/**
+FILE: SampleAuthentication.cs
+DESCRIPTION:
+    This sample demonstrates how to authenticate to the Document Translation service.
+    1) Use a Document Translation API key with AzureKeyCredential from azure.core.credentials
+    Note: the endpoint must be formatted to use the custom domain name for your resource:
+    https://<NAME-OF-YOUR-RESOURCE>.cognitiveservices.azure.com/
+USAGE:
+Set the environment variables with your own values before running the sample:
+    1) AZURE_DOCUMENT_TRANSLATION_ENDPOINT - the endpoint to your Document Translation resource.
+    2) AZURE_DOCUMENT_TRANSLATION_KEY - your Document Translation API key
+**/
+
+namespace DocumentTranslatorSamples
+{
+    public class SampleAuthentication
```

Comment:
In general, the way the samples work in .NET is in two ways:
- documentation samples: They are in the root of the folder and it is what we want people to look at. They are .md files that include more explanations and follow a structure. You can see them here: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/translation/Azure.AI.Translation.Document/samples
- Code samples: They are runnable during our tests, so we make sure our samples work. We take abstracts or snippets from here and feed the samples .md files. Usually, these ones have extra configuration and show people how to make a full call. We provide sync and async samples. Example here: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/translation/Azure.AI.Translation.Document/tests/samples

With that in mind, we always start all code samples with authentication, and we also start all of our .md samples with how to authenticate to the client. i.e. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/translation/Azure.AI.Translation.Document/samples/Sample1_StartTranslation.md#creating-a-documenttranslationclient

Because of that, I don't think this sample is necessary, but if we want to leave it, I am also ok with it. 
What do you think?

Reply:
Removed it. Thanks for pointing to the test samples.